torch/csrc/jit/runtime/operator.h

```diff
@@ -69,7 +69,7 @@ struct TORCH_API Operator {
 
   Operator(
       const std::string& schema,
-      int (*op)(Stack&),
+      Operation op,
```

Comment:
iirc, @zdevito was the one that made this constructor a non-capturing function pointer on purpose. @zdevito: can you take a look at this part of the file?

Reply:
Yeah, this change is intentional. We don't want simple operators to store state because it makes it hard in the future to optimize the execution if we have to assume everything is more than a raw pointer.

Reply:
I need to undo this, it's no longer necessary.